### ISSUE TITLE
[BUG] Description

### ISSUE THREAD

**Issue by benwiley4000** [OpenAPITools/openapi-generator#6423]
#### Bug Report Checklist

- [X] Have you provided a full/minimal spec to reproduce the issue?
- [N/A] Have you validated the input using an OpenAPI validator ([example](https://apidevtools.org/swagger-parser/online/))?
- [X] What's the version of OpenAPI Generator used?
    5.0.0-SNAPSHOT
- [X] Have you search for related issues/PRs?
- [X] What's the actual output vs expected output?
- [ ] [Optional] Bounty to sponsor the fix ([example](https://www.bountysource.com/issues/66123212-javascript-client-produces-a-wrong-object-for-a-string-enum-type-that-is-used-with-ref))

##### Description

I'm using the typescript-axios generator with the `withSeparateModelsAndApi` flag:

```yml
withSeparateModelsAndApi: false
apiPackage: 'api'
modelPackage: 'model'
```

Basically, when using this, the models aren't exported from the root package:

```ts
// index.d.ts
export * from "./api";
export * from "./configuration";
```

However both the API classes and models are correctly included with the default setup which uses a single file for everything. It would be desirable to export the models in all cases. The separate-file structure is much nicer for editors that are slow processing large files.

##### openapi-generator version

5.0.0-SNAPSHOT

##### OpenAPI declaration file content or url

See above

##### Suggest a fix

Just export the `modelPackage` directory as well, don't think it's super complicated.

**Comment by hleb-albau** [comment]
same behavior

**Comment by calvinwyoung** [comment]
@benwiley4000 PR #6873 was just merged to fix this — you should be able to close this issue now.
